src/main/java/com/cloudbees/groovy/cps/Env.java

```diff
@@ -1,21 +1,56 @@
 package com.cloudbees.groovy.cps;
 
+import com.cloudbees.groovy.cps.impl.LocalVariableBlock;
+import com.cloudbees.groovy.cps.impl.VariableDeclBlock;
 import com.cloudbees.groovy.cps.sandbox.Invoker;
 import groovy.lang.Closure;
 
+import javax.annotation.Nonnull;
 import java.io.Serializable;
 import java.util.List;
 
 /**
- * For variable lookup. This is local variables.
+ * Represents an environment in which {@link Block} is evaluated.
```

Comment:
Worth mentioning that this would be just the defined, named variables (since sometime environment can refer to a lot more about the execution environment and internal state, much like with contexts).